### DISCUSSION TITLE
Way to easily recognise wich objects contain Script

### DISCUSSION THREAD

**Discussion by Miurath** [eclipse-birt/birt#973] [General]
Hi, 
id like to know if there is a way to show wich objects contain script. This feature would be very helpful, since finding all the script in an old .rptdesign that was written by a diffrent person, is a pain.
Is there maybe an icon that can be shown in the Outline or a summary for this?

**Comment by hvbtup** [comment]
The very last point in the outline is "Scripts":
![grafik](https://user-images.githubusercontent.com/7644776/169337735-6f63040e-688a-4619-90df-76bdf6528918.png)

So it is in fact easy to find all the scripts.

But - as I have been using Oracle Reports for decades - I have to admit that it was easier to see there. Objects with scripts had a little symbol "P" (for PL/SQL) as icon overlay (similar to what BIRT does for objects which are referenced from a library).

I think an additional icon overlay (say: "S" for Script) _could_ be added, but I'm -1 for it: Since we already have the string symbol for referenced objects, this could easily become messy.
